lib/autoupdate/agent/installer.go

```diff
@@ -430,11 +430,18 @@ func (li *LocalInstaller) Link(ctx context.Context, rev Revision, force bool) (r
 	return revert, nil
 }
 
+func emptyRevert(_ context.Context) bool {
+	return true
+}
+
 // LinkSystem links the system (package) version into LinkBinDir and CopyServiceFile.
 // LinkSystem returns ErrInvalid if LinkBinDir is not DefaultLinkDir.
```

Comment:
Nit: explain that this prevents namespaced installs from trying to link the system install back.